src/features/email/services/email-data.ts

```diff
@@ -7,18 +7,19 @@ export const emailData: TEmail[] = [
     subject: 'Meeting Tomorrow',
     preview:
       "Hi, Let's have a meeting tomorrow to discuss the project. I've been reviewing the project details and have some ideas I'd like to share. It's crucial that we align on our next steps to ensure the project's success. Please come prepared with any questions or insights you may have. Looking forward to our meeting! Best regards, Adrian",
-    date: '26.12.2024',
+    date: 'Sun Mar 23 2025 15:31:25 GMT+0600 (Bangladesh Standard Time)',
```

Comment:
It's better to use the date value as ISO date time format, since later if you need to implement the backend API then they will give you only ISO datetime format